layer/contracts/projects.py

```diff
@@ -132,6 +132,18 @@ def with_readme(self, readme: Optional[str]) -> "Project":
     def with_functions(self, functions: Sequence[Function]) -> "Project":
         return replace(self, functions=functions)
 
+    def __str__(self) -> str:
+        from layer.config import DEFAULT_PATH, ConfigManager
+
```

Comment:
maybe we can read the config only if there is an account ?

Reply:
Done